src/main/java/frc/robot/autonomous/BinTrajectoryParameterRuntimeTest.java

```diff
@@ -0,0 +1,52 @@
+package frc.robot.autonomous;
+
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.kinematics.SwerveDriveKinematics;
+import edu.wpi.first.math.trajectory.Trajectory;
+import edu.wpi.first.math.trajectory.TrajectoryConfig;
+import edu.wpi.first.math.trajectory.TrajectoryGenerator;
+import frc.robot.Constants.Drive.Dims;
+import frc.robot.autonomous.Waypoints.OnsideStartToInnerCargoAndBack;
+import java.util.List;
+
+public class BinTrajectoryParameterRuntimeTest {
```

Comment:
Should this make it to dev @vecless ? Not sure what it does but I remember it was important

Reply:
Oh it's just a test program to evaluate how long a trajectory takes to run. I'll break it out of the main code

Reply:
sounds good